code/style.css

```diff
@@ -1,4 +1,297 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box;
+  }
+
+.body {
+  background:linear-gradient(#cbf7f7, #2d73d4, #020f3f, #020f3f,black, black, black);
+  color:rgb(226, 230, 231);
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 16px;
+  letter-spacing: 0.07rem;
+  min-height: 3000px;
 }
+
+/* HEADER */
+
+
+header {
+  display: flex;
+  background: none;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 30px 10px 10px;
+  height: 90px;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.logo {
+  animation: pulse 7s linear infinite;
+  height: 80px;
+}
```

Comment:
I really like this animation! Just make sure that your header content fits on all widths:
<img width="332" alt="Skärmavbild 2021-02-03 kl  13 40 43" src="https://user-images.githubusercontent.com/51529201/106748969-448fd780-6626-11eb-8832-5ecfb0ea5969.png">